### ISSUE TITLE
Elixir Tools, Brackets Elixir plugins don't work. Kindly help to get one or other work

### ISSUE THREAD

**Issue by rokfrog** [adobe/brackets#14525]
### Prerequisites

* [ ] Can you reproduce the problem with `Debug -> Reload Without Extensions`? Yes
* [ ] Did you perform a cursory search to see if your bug or enhancement is already reported? No
* [ ] Did you read the [Troubleshooting guide](https://github.com/adobe/brackets/wiki/Troubleshooting)?

For more information on how to write a good bug report read [here](https://github.com/adobe/brackets/wiki/How-to-Report-an-Issue)
For more information on how to contribute read [here](https://github.com/adobe/brackets/blob/master/CONTRIBUTING.md)

### Description
I'm new to Brackets, needed some IDE which is light and works for Elixir, but both the plugins Elixir Tools and Brackets Elixir doesn't work. Kindly help.
[Description of the bug or feature]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expected to happen]

**Actual behavior:** [What actually happened]

### Versions

Please include the OS and what version of the OS you're running.
Please include the version of Brackets. You can find it under `Help -> About Brackets` (Windows and Linux) or `Brackets -> About Brackets` (macOS)

**Comment by navch** [comment]
Looks like both these extensions are quite old. Could you please log an issue on those Extension's repo.

**Comment by rokfrog** [comment]
Yes, they are pretty old. I did for Brackets Elixir and have to do it for Elixir tools too. Brackets is lightweight, and it just needs these plugins just like vscode-elixir with a perfect IDE terminal. That would be best option for Elixir and Phoenix than slightly heavy VScode., so thought to report it in here in case Bracket team themselves come up something with that

**Comment by navch** [comment]
Great. Let them(owners of Elixir tools/Brackets Elixir) come back on this issue. Please add a reference to that issue here.
